### PR TITLE
Load the Template class in case it's not loaded

### PR DESCRIPTION
**Pull request by nickvergessen**
Should backport to 10, so access control can have it's own admin section there.

### CODE REVIEW

File: apps/workflowengine/lib/AppInfo/Application.php [hunk 1]
```diff
@@ -41,32 +40,27 @@ public function registerHooksAndListeners() {
...
-					'systemtags/systemtagmodel',
-					'systemtags/systemtagscollection',
-				]);
+				Util::addScript('core', 'oc-backbone-webdav');
+				Util::addScript('core', 'systemtags/systemtags');
+				Util::addScript('core', 'systemtags/systemtagmodel');
+				Util::addScript('core', 'systemtags/systemtagscollection');
 
-				vendor_script('jsTimezoneDetect/jstz');
+				\OC_Util::addVendorScript('jsTimezoneDetect/jstz');
```
Comment (LukasReschke) [inline apps/workflowengine/lib/AppInfo/Application.php]: line   50: OC_Util - Static method of private class must not be called

Comment (nickvergessen) [inline apps/workflowengine/lib/AppInfo/Application.php]: Well the only way I then see is to add that method to `OCP\Util`, or to load the template file somehow

Comment (LukasReschke) [inline apps/workflowengine/lib/AppInfo/Application.php]: Also related to https://github.com/nextcloud/server/issues/889

**Comment by blizzz** [comment]
Works for me in conjunction with https://github.com/nextcloud/files_accesscontrol/pull/22

:+1:

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Pull request merged**